streams/cr.m3u

```diff
@@ -65,6 +87,10 @@ http://tiquiciatv.com:1935/stv/web/playlist.m3u8
 https://lstv.duckdns.org:449/hls/stv.m3u8
 #EXTINF:-1 tvg-id="SOYPlanchaTV.cr",SOY Plancha TV (720p)
 https://59ef525c24caa.streamlock.net/vmtv/soyplancha/playlist.m3u8
+#EXTINF:-1 tvg-id="TDtv.cr",TDtv (720p)
+https://cloudvideo.servers10.com:8081/8230/index.m3u8
+#EXTINF:-1 tvg-id="TeleBrunca.cr",TeleBrunca (720p)
+https://59ef525c24caa.streamlock.net/telebrunca/telebrunca/playlist.m3u8
```

Comment:
404

Reply:
Also doesn't work on its official site, so i think is Not 24/7